### ISSUE TITLE
Cluster health/status inconsistent between Tendrl UI and Cluster Dashboard (which mirrors gstatus)

### ISSUE THREAD

**Issue by julienlim** [Tendrl/ui#673]
Cluster health/status in Tendrl UI inconsistent between Tendrl UI and Cluster Dashboard (which mirrors gstatus)

Using gstatus and Grafana dashboard, it says Cluster is Unhealthy (as 1 of the nodes is down).  Tendrl UI says it's healthy

![screen shot 2017-10-09 at 9 33 53 am](https://user-images.githubusercontent.com/5589919/31347678-1a10ddb2-aceb-11e7-8b3f-f4ce604ea482.png)

![screen shot 2017-10-09 at 12 12 46 pm](https://user-images.githubusercontent.com/5589919/31347691-2716d87c-aceb-11e7-8db3-0dc4ca53a159.png)

![screen shot 2017-10-09 at 9 32 24 am](https://user-images.githubusercontent.com/5589919/31347680-1ca017fa-aceb-11e7-8897-519cf1bac4ca.png)

@a2batic @gnehapk @nthomas-redhat @Tendrl/tendrl-qe @mcarrano

**Comment by julienlim** [comment]
Interestingly, when I brought down the entire Gluster cluster, the Tendrl UI still said the cluster was healthy.

<img width="1172" alt="screen shot 2017-10-09 at 4 28 53 pm" src="https://user-images.githubusercontent.com/5589919/31357294-26dbc68c-ad0f-11e7-865c-4ec7c98502ba.png">

**Comment by gnehapk** [comment]
@julienlim Please share the API response of "/clusters" received for it.

**Comment by gnehapk** [comment]
@julienlim Fixed with the latest build. Please verify.

**Comment by julienlim** [comment]
@gnehapk @Tendrl/tendrl-qe  I'm noticing the red x on the cluster when it's unhealthy (in Tendrl UI).  I think we need to use a different icon to indicate degraded / unhealthy as the red x can be alarming to the user.

@mcarrano - can you please suggest an icon?

I did notice that node4 (which is down to create this unhealthy state) is missing the down icon.

See screenshots below:

<img width="1131" alt="screen shot 2017-10-11 at 2 02 49 pm" src="https://user-images.githubusercontent.com/5589919/31458459-2dfa4daa-ae8e-11e7-912d-f3881ccf52f0.png">

<img width="1133" alt="screen shot 2017-10-11 at 2 07 09 pm" src="https://user-images.githubusercontent.com/5589919/31458619-a0380bc8-ae8e-11e7-822d-f86df9a25293.png">

<img width="1133" alt="screen shot 2017-10-11 at 2 15 34 pm" src="https://user-images.githubusercontent.com/5589919/31458645-ae3935e4-ae8e-11e7-9e42-edb30850a3a6.png">

**Comment by julienlim** [comment]
@gnehapk @cloudbehl @Tendrl/tendrl-qe 

In chatting with @mcarrano, we decided that we should show the status icon for the Cluster Unhealthy status as  'pficon-warning-triangle' in orange.

We also need to synch this with the grafana dashboards, so that the degraded / unhealthy cluster is in orange (as red is down).

<img width="1205" alt="screen shot 2017-10-11 at 2 42 39 pm" src="https://user-images.githubusercontent.com/5589919/31460031-0231d81e-ae93-11e7-9839-04d7b0a9fab6.png">

**Comment by a2batic** [comment]
@julienlim, changed the icon of Unhealthy Cluster as 'pficon-warning-triangle' in tendrl-ui-1.5.3-20171025T055918.53fc027.noarch.rpm. Please verify.

**Comment by julienlim** [comment]
Noting that collectd default interval is 3 min (or 180seconds) per looking at /etc/collectd.conf.

Observed that when node brought down, it takes UI approximately 3 minutes to notice the node is down (and Grafana dashboard still doesn't show node is down), and then another 3 minutes before the dashboards reflect the node is down, thus 6 minutes for dashboard to reflect node down.

Noting that monitoring status seems to show a significant lag time.

@Tendrl/tendrl-qe @gnehapk @mcarrano

**Comment by gnehapk** [comment]
@julienlim Host list is getting polled after 30 secs, so the updated data should be reflected in UI by max 30 secs(if it is getting reflected in the API properly). Can you please share the API response received for "/nodes" on host list page once the collectd default interval is finished.

**Comment by julienlim** [comment]
Noting my [previous comment](https://github.com/Tendrl/ui/issues/673#issuecomment-339443966), that the synchronization delay is related to https://github.com/Tendrl/api/issues/306 issue.

@gnehapk I've since redeployed my Tendrl deployment and cannot provide the API response for /nodes.

From referring to https://github.com/Tendrl/api/issues/306, the 3 min delay of updating the Tendrl UI is likely due to the default "sync_interval" specified in node-agent.conf.yaml as 180 seconds, or 3 min.  Since collectd default is 3 min or another 180 seconds, you then see the update in Grafana dashboards depending how the intervals line up.

Going back to this specific issue of the Cluster health/status inconsistent between Tendrl UI and Cluster Dashboard, this has been resolved (and therefore can be closed), though the synchronization delay is still cause for concern, but since it's already mentioned in https://github.com/Tendrl/api/issues/306, we can close this issue.

@Tendrl/tendrl-qe @fbalak @r0h4n @mcarrano
